addons/pos_restaurant/static/src/app/pos_store.js

```diff
@@ -25,15 +25,15 @@ patch(PosStore.prototype, "pos_restaurant.PosStore", {
      * @override
      */
     async setup() {
+        this.orderToTransfer = null; // table transfer feature
+        this.transferredOrdersSet = new Set(); // used to know which orders has been transferred but not sent to the back end yet
+        this.floorPlanStyle = "default";
+        this.isEditMode = false;
```

Comment:
Is there a specific reason for this change?

Reply:
There is. In super, I call _createOrder(json) in the setup. This method is overriden in pos_restaurant and needs some of the above fields and as it does not change the behaviour of the rest, I decided to put it there, before calling super